python/ray/data/datasource/parquet_datasource.py

```diff
@@ -205,10 +205,10 @@ def __init__(
 
             filtered_paths = set(expanded_paths) - set(paths)
             if filtered_paths:
-                logger.info(f"Filtered out the following paths: {filtered_paths}")
-
-        if len(paths) == 1:
-            paths = paths[0]
+                logger.info("Filtered out %d paths", len(filtered_paths))
+        else:
+            if len(paths) == 1:
+                paths = paths[0]
```

Comment:
Is there any case where we'd want to unwrap the path? If not, we can probably just remove this case?

Reply:
It turns out a lot of tests failed. Leaving this here.